cm_bpyNodes.py

```diff
@@ -478,7 +478,7 @@ class FilterNode(LogicNode):
 
     def draw_buttons(self, context, layout):
         layout.prop(self, "Operation")
-        if (self.Operation == "EQUAL" or self.Operation == "NOT EQUAL" or self.Operation == "LESS" or self.Operation == "GREATER"):
+        if self.Operation in ["EQUAL", "NOT EQUAL", "LESS", "GREATER"]:
```

Comment:
I got a quick answer :-) 

Looks like the "in" is fine...will change to a dict though for speed.